warehouse_models.py

```diff
@@ -6,4 +6,6 @@ class Item(ndb.Model):
     updated = ndb.DateTimeProperty(auto_now_add=True)
     name = ndb.StringProperty(required=True)
     description = ndb.StringProperty(required=False)
-    qr_code = ndb.IntegerProperty(required=False)
\ No newline at end of file
+    qr_code = ndb.IntegerProperty(required=False)
+    type = ndb.StringProperty(required=True)
```

Comment:
Would it be okay to change this name, maybe to "item_type"? I only ask since "type" is a Python keyword.

Reply:
I considered that issue too. I'll change it to item_type